pkg/envoy/xds/server.go

```diff
@@ -309,13 +313,19 @@ func (s *Server) processRequestStream(ctx context.Context, streamLog *logrus.Ent
 			}
 
 			index, exists := typeIndexes[typeURL]
-
 			if !exists {
 				requestLog.Error("unknown type URL in xDS request")
 				return ErrUnknownTypeURL
 			}
 
 			state := &typeStates[index]
+			watcher := s.watchers[typeURL]
+
+			nodeIP, err := IstioNodeToIP(req.GetNode())
```

Comment:
Should this `IstioNodeToIP` function be renamed?

Reply:
Not really, it is the Istio convention for structuring the Envoy Node ID that we use to be able to derive the IP address of the node. We control the node ID format for the host proxy ourselves, so using the Istio format also for the host proxy allows more uniform handling of the node ID.